src/crafting_gui.cpp

```diff
@@ -424,6 +426,157 @@ static input_context make_crafting_context( bool highlight_unread_recipes )
     return ctxt;
 }
 
+class recipe_result_info_cache
+{
+        std::vector<iteminfo> info;
+        const recipe *last_recipe = nullptr;
+        int last_width = 0;
+
+        void get_byproducts_data( const recipe *rec, const int count, std::vector<iteminfo> &summary_info,
+                                  std::vector<iteminfo> &details_info );
+        void get_item_details( const item &dummy_result, const int count,
```

Comment:
Note the new error (sry for spam, if you notice the errors), the declaration and definition have different parameter names.

Reply:
I did see the error.  This was a rather silly one, but it should be fixed now.  I could have sworn I'd run clang-tidy on my end before pushing that commit.
I'm not sure what was causing the other failure (General build matrix / Clang 12, Ubuntu, Tiles, ASan (pull_request) ).  I don't _think_ it's on my end.